src/services/SearchService.ts

```diff
@@ -104,6 +111,27 @@ async function searchCsv(entity: RawSearchRequestPayload): Promise<any> {
   }
 }
 
+function convertToPhraseMatch(searchNode: SearchNodePayload): SearchNodePayload {
```

Comment:
But I think it is worth reviewing this function. Likely need to export and add tests as well.